example/glue/CollectorLanguageInterfaceImpl.cpp

```diff
@@ -192,7 +221,8 @@ MM_CollectorLanguageInterfaceImpl::scavenger_masterSetupForGC(MM_EnvironmentBase
 void
 MM_CollectorLanguageInterfaceImpl::scavenger_workerSetupForGC_clearEnvironmentLangStats(MM_EnvironmentBase *env)
 {
-	/* Do nothing for now */
+	MM_EnvironmentStandard *envStd = (MM_EnvironmentStandard *)env;
+	MM_SublistFragment::flush((J9VMGC_SublistFragment*)&envStd->_scavengerRememberedSet);
```

Comment:
I think this is the right place but the containing method name is too specific. I had intended to come back to this and rename this method to scavenger_workerSetupForGC(), an analogue for scavenger_masterSetupForGC(). The scavenger_workerSetupForGC() method/stub provides a place for threads that are participating in a GC to address language-specific concerns prior to starting GC. This includes flushing thread-local remembered set buffers. I was tempted to just insert a call to MM_Scavenger::flushRememberedSet(env) in MM_Scavenger::workerSetupForGC() but held off, thinking that would have been done already if necessary (for Java).

I would like to try calling MM_Scavenger::flushRememberedSet(env) from MM_Scavenger::workerSetupForGC() and removing the flush from scavenger_workerSetupForGC_clearEnvironmentLangStats(). All of this (remembered set management) should be OMR-generic code. Also suggest replacing CLI::scavenger_workerSetupForGC_clearEnvironmentLangStats() with CLI::scavenger_workerSetupForGC(). 

Let's discuss in the morning.